PencilCase/PencilCase/scripts/Repositories/ProductRepository.ts

```diff
@@ -0,0 +1,64 @@
+﻿import { Application } from '../application';
+import { Product } from '../Models/Product';
+
+export class ProductRepository {
+    private db: Database;
+
+    constructor() {
+        this.db = Application.instance.openDataBase();
+    }
+
+    public getAll = (successCallback?: (transaction: SqlTransaction, resultSet: SqlResultSet) => void, errorCallback?: (transaction: SqlTransaction, sqlError: SqlError) => void) => {
+        this.db.transaction((transaction) => {
+            transaction.executeSql('select P.*, UOM1.Name RetailUnitName, UOM2.Name WholesaleUnitName from Product P \
```

Comment:
While getting all products, added 2 extra column to represent the retail and wholesale unit name by inner join syntax.